doc/news/_preparation_next_release.md

```diff
@@ -40,27 +40,27 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.8.
   The `mmapstorage_crc` variant enables CRC32 checksums for critical data,
   while the `mmapstorage` variant omits the checksum for maximum performance.
 
-  We ran a synthetic benchmark with 127 iterations using 40k keys in a keyset,
+  We ran a synthetic benchmark with 257 iterations using 40k keys in a keyset,
```

Comment:
Did you measure the time of all 257 iterations together?

Why increasing this number? 127 is already quite high, usually the iterations are only used to avoid that an outlier is taken.

Reply:
For the average times I provided, I give the median of 257 measured instances. I did not sum it up, if that is what you mean.

I wanted to get rid of some outliers. I found later that it was because of the turbo boost and cpu freq governor of my cpu. For the updated numbers, turbo was disabled and cpu frequency governor was set to "performance".